src/zone.rs

```diff
@@ -64,6 +65,11 @@ pub struct Zone {
     )]
     pub boundary: Option<geo::MultiPolygon<f64>>,
 
+    #[serde(
+        serialize_with = "serialize_bbox_as_geojson", deserialize_with = "deserialize_as_bbox"
```

Comment:
hum can you test to load an old file with cosmogony ?
I think you should also add a serde's `default`